address-sanitizer/asan-glibc/asan-glibc-build.sh

```diff
@@ -39,7 +39,7 @@ build_asan() {
   gcc -c asan-init-stub.c -o asan-init-stub.o -fPIC
  
   python -m compileall $HERE 2>/dev/null
-  chmod +x *.pyc
+  chmod +x ./*.pyc
```

Comment:
why this one is needed?

Reply:
Use ./*glob* or -- *glob* so names with dashes won't become options.
https://github.com/koalaman/shellcheck/wiki/SC2035